src/openrct2-ui/interface/Graph.cpp

```diff
@@ -26,10 +26,9 @@ static void graph_draw_months_uint8_t(
         if (history[i] != 255 && yearOver32 % 4 == 0)
         {
             // Draw month text
-            auto ft = Formatter::Common();
+            auto ft = Formatter();
             ft.Add<uint32_t>(DateGameShortMonthNames[date_get_month((yearOver32 / 4) + MONTH_COUNT)]);
-            gfx_draw_string_centred(
-                dpi, STR_GRAPH_LABEL, screenCoords - ScreenCoordsXY{ 0, 10 }, COLOUR_BLACK, gCommonFormatArgs);
+            gfx_draw_string_centred(dpi, STR_GRAPH_LABEL, screenCoords - ScreenCoordsXY{ 0, 10 }, COLOUR_BLACK, ft.Data());
```

Comment:
You could use `DrawTextBasic`, so that you can pass `ft` directly.

Reply:
I thought about that as well and sort of came to the same conclusion. I think there is less chance of error doing this in two steps